client/components/Item/Metadata/MetadataEditor.vue

```diff
@@ -19,7 +19,7 @@
             <v-text-field
               v-model="metadata.Name"
               outlined
-              :label="$t('title')"
+              :label="$t('item.tracklist.title')"
```

Comment:
I think the metadata editor shouldn't depend on the track list. This thing needs to essentially be entirely rewritten anyway.

I know you did this because currently the key doesn't exist, but something like `item.metadata.title` is probably better, if only because the term used in the track list and the metadata editor might be different between languages depending on context.

Plus we can group translations together here (which we can't in jf-web) so it's less of an issue for organization. Weblate can also help a lot if we handle rules and the glossary properly.